design/video-infrastructure.md

```diff
@@ -2,49 +2,49 @@
+Once we get a new stream request, we need to figure out who went live. What channel is the corresponding stream request for (authentication), and what features does the requested stream have access to?
 
-We make a API call to the auth server to validate the incoming stream and then we make some API calls to create a stream target.
+We make an API call to the auth server to validate the incoming stream and then make some API calls to create a stream target.
 
-Once everything passes and we are all ready and set, we ingest the stream and convert it to a common protocol (SRT for example)
+Once everything passes, and we are all ready and set, we ingest the stream and convert it to a standard protocol (SRT, for example)
 
-At this point we can also pass the stream to other optional or future ideas (such as transcription or image detection).
+We can pass the stream to other optional or future ideas (such as transcription or image detection).
 
 We then push the newly ingested stream onto the next stage: Transcoding.
 
 ## Transcoding
 
-In the transcoding stage we fetch the stream from the ingest stage and then start creating video variants from it.
+In the transcoding stage, we fetch the stream from the ingest location and create video variants.
 
-We can transcode it down into `1080p`, `720p`, `480p` or lower depending on what the specific requirements and features the stream has.
+We can transcode it down into `1080p`, `720p`, `480p`, or lower depending on the stream's specific requirements and features.
 
-Transcoding is important since we can significantly reduce the bandwidth cost of the video by compressing it using codecs such as [H264](https://en.wikipedia.org/wiki/Advanced_Video_Coding), [VP8](https://en.wikipedia.org/wiki/VP8), [VP9](https://en.wikipedia.org/wiki/VP9) or [AV1](https://en.wikipedia.org/wiki/AV1). 
-We can also decrease the image quality for viewers with slower connections or smaller displays. If the viewer has a slower connection we still want to provide them a stream to watch, and to do this we need to be able to provide a lower quality stream for them. Or if they have a smaller display providing a higher resolution stream does not make any difference and can in some cases look worse.
+Transcoding is crucial since we can significantly reduce the bandwidth cost of the video by compressing it using codecs such as [H264](https://en.wikipedia.org/wiki/Advanced_Video_Coding), [VP8](https://en.wikipedia.org/wiki/VP8), [VP9](https://en.wikipedia.org/wiki/VP9) or [AV1](https://en.wikipedia.org/wiki/AV1). 
+We can also decrease the image quality for viewers with slower connections or smaller displays. If the viewer has a slower connection, we still want to provide them with a stream to watch and to do this. We need to be able to provide a lower-quality stream for them. Or if they have a smaller display providing a higher resolution stream does not make any difference and can sometimes look worse.
```

Comment:
"provide them with a stream to watch and to do this. We need..." a period literally breaking the whole structure. Should be something else, "to do this we need" should be joined somehow.

Reply:
replace the period with a semi colon `;`